sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -631,7 +633,7 @@ def get_media_settings_value(physical_port, key):
             if len(default_dict) != 0:
                 return default_dict
             else:
-                helper_logger.log_error("Error: No values for physical port '{}'".format(physical_port))
+                helper_logger.log_info("Error: No values for physical port '{}'".format(physical_port))
```

Comment:
why not a Error level log?

Reply:
On SKUs which have media_settings.json file present, but have ports with transceivers not requiring NPU SI settings, we see this error.
Hence, to handle such scenario, I am planning to change this to info.